### PR TITLE
Adds explanation for .arb file

### LINKED ISSUE #7627: Explain `.arb` file type on 'Internationalizing Flutter apps' page
### Page URL

https://docs.flutter.dev/development/accessibility-and-localization/internationalization/

### Page source

https://github.com/flutter/website/tree/main/src/development/accessibility-and-localization/internationalization.md

### Describe the problem

The page mentions `.arb` files, but doesn't clarify what the format it is.

### Expected fix

Add a parenthesis after the first mention of `.arb` file that says ARB file is Google's App Resource Bundle, and gives a link to https://github.com/google/app-resource-bundle.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #7627

### CODE REVIEW

File: src/accessibility-and-localization/internationalization.md [hunk 1]
```diff
@@ -1229,3 +1234,4 @@ Rebuilding `l10n/messages_all.dart` requires two steps.
 [widgets-local]: {{site.api}}/flutter/widgets/Localizations-class.html
 [widgets-global]: {{site.api}}/flutter/flutter_localizations/GlobalWidgetsLocalizations-class.html
 [`WidgetsApp`]: {{site.api}}/flutter/widgets/WidgetsApp-class.html
+[App Resource Bundle]: https://github.com/google/app-resource-bundle
```
Comment (sfshaza2) [inline src/accessibility-and-localization/internationalization.md:1237]:
```suggestion
[App Resource Bundle]: {{site.github}}/google/app-resource-bundle
```

**Review by sfshaza2 — COMMENTED** [review]
lgtm with one minor nit. Tony, you must run tool/refreshcodeexcerpts when you modify a code excerpt, even if it's just a comment.

**Comment by atsansone** [comment]
@sfshaza2 : Thanks for the tip!

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**